server/stream.go

```diff
@@ -104,6 +104,49 @@ type StreamConfig struct {
 	Metadata map[string]string `json:"metadata,omitempty"`
 }
 
+// clone performs a deep copy of the StreamConfig struct, returning a new clone with
+// all values copied.
+func (cfg *StreamConfig) clone() *StreamConfig {
+	clone := *cfg
+
```

Comment:
Small nit but lets remove all the blank lines..

Reply:
Done

Reply:
Thanks!